4001-5000/LIT4551Salam.xml

```diff
@@ -2,7 +2,7 @@
     <teiHeader xml:base="https://betamasaheft.eu/">
         <fileDesc>
             <titleStmt>
-                <title xml:lang="en" xml:id="t1">Salām-hymn to Lālibalā</title>
```

Comment:
Do you avoid the title "Salām-hymn" on purpose?

Reply:
Yes, but I am fine with reintroducing it. My first association with salām-hymn is the Synaxary salām-hymns, which are something different from the one here.

Reply:
Without arguing that my 'name is good, I referred to it as ‘Salāmta as edited in GQL’, for is appears in various forms but was edited as 'Salāmta' by the community of Lalibala.

Reply:
Thank you!
I think there is no need for a unique title, as we have the CAe reference for that. Any identifying information can of course be supplied in the abstract, or, in the case of an edition, in the bibliography. Should you have the references at hand at some point, it would be of course wonderful to insert them.
Is "Salāmtā" the modern Amharic equivalent of the Ethiopianists' "Salām-hymn"? Then I would stick for now with "salām-hymn" if "hymn" is rejected, which is more commonly, though not always clearly, used.

Reply:
Dear Dorothea, apologies for not following the rules of bm politeness! Thank you! In this case whatever choice you decide to make - whether Salām-hymn or Salām-hymn for Lalibala, I have no feelings against.

Reply:
There are no rules :) It is the question between "Salām-hymn to Lālibalā" and "Hymn to Lālibalā", no intention of hiding his name from the readers.

Reply:
Dear Dorothea, I see! But then, if Denis does not mind, I personally have no objections against 'Hymn to Lālibalā', as the incipit will reveal its salām nature :-)